runtime/src/integration/bitcoin_simulator.rs

```diff
@@ -240,19 +241,32 @@ impl BitcoinCoreApi for MockBitcoinCore {
             sleep(Duration::from_secs(1)).await;
         }
     }
+    async fn get_balance(&self, min_confirmations: Option<u32>) -> Result<Amount, BitcoinError> {
+        Ok(Amount::ZERO)
```

Comment:
Why not keep the `unimplemented!()`?

Reply:
I think the tests were failing because of it